src/geo/ui/widgets/widget-view.js

```diff
@@ -3,26 +3,42 @@ var WidgetLoaderView = require('./standard/widget_loader_view');
 var WidgetErrorView = require('./standard/widget_error_view');
 
 /**
- *  Default widget view:
+ * @classdesc Default widget view:
  *
- *  It contains:
- *  - view model (viewModel)
- *  - data model (dataModel)
+ * It contains:
+ * - view model (viewModel)
+ * - data model (dataModel)
  *
  *  It will offet to the user:
  *  - get current data (getData)
  *  - Sync or unsync widget (sync/unsync), making the proper view
  *  listen or not changes from the current datasource.
+ *
+ * @class geo.ui.widgets.WidgetView
```

Comment:
That explanation corresponds to that class and IMO is good to have it at the same file. Of course we can also document each namespace describing briefly the goal of the set of classes, types, etc it contains.